cookies/views/resource.py

```diff
@@ -886,19 +886,23 @@ def dataset(request, dataset_id):
     Details about a dataset.
     """
     dataset = get_object_or_404(Dataset, pk=dataset_id)
+    resource_param = None
```

Comment:
resource_param is a bad variable name. what does that mean?

Reply:
Changed the variable name to source_resources